docs/specviz/import_data.rst

```diff
@@ -40,16 +40,16 @@ Here, users can select the loaded data set to be visualized in the viewer.
 Loading data via the API
 ------------------------
 Alternatively, if users are working in a coding environment like a Jupyter notebook, they have access to the :class:`~jdaviz.configs.specviz.helper.Specviz` helper class API. Using this API, users can load data into the application through code.
-Below is an example of importing the :class:`~jdaviz.configs.specviz.helper.Specviz` helper class, creating a :class:`~specutils.Spectrum1D` object from a data file via the :func:`~specutils.Spectrum1D.read` method::
+Below is an example of importing the :class:`~jdaviz.configs.specviz.helper.Specviz` helper class, creating a :class:`~specutils.Spectrum1D` object from a data file via the `specutils.Spectrum1D.read` method::
```

Comment:
@rosteen pointed out that `specutils.Spectrum1D.read` is listed as "attribute" upstream even though it is a method. 🤷